localstack-core/localstack/services/dynamodb/provider.py

```diff
@@ -752,7 +752,8 @@ def describe_table(
             if replica_region != context.region:
                 replica_description_list.append(replica_description)
 
-        table_description.update({"Replicas": replica_description_list})
+        if replica_description_list:
+            table_description.update({"Replicas": replica_description_list})
```

Comment:
nit: I think we could do `table_description["Replicas"] = replica_description_list` instead, but it was that way before so let's keep it, only a nit anyway 😄